storyblok/storyblok-design-system
# refactor: sb menu

## Pull request type

Jira Link: [INT-](url)




## How to test this PR


Compare:

- https://blok.ink/?path=/docs/design-system-components-sbmenu--default
- https://storyblok-design-system-git-refactor-ar-24-706e51-storyblok-com.vercel.app/path=/docs/design-system-components-sbmenu--default


It's also good to check in the app:
```cd storyblok-design-system && git checkout refactor/ar-24-sb-menu```
Then start the server of the app


## What is the new behavior?



- refactors the menu component to use the template api
- 
- 

## Other information
